README.md

```diff
@@ -40,6 +41,14 @@ For example: `ci/django.yml` and `ci/properties/django.properties.json`.
 * monitoring
 * Automation
 * utilities
+* Pages
+* Hugo
+* Gatsby
+* Next
+* Nuxt
+* HTML
+* JavaScript
+* CSS
```

Comment:
I think we don't need these languages/frameworks in categories. `Pages` seems enough

Reply:
These languages/frameworks are covered in `Valid properties` section under categories

Reply:
Now only the frameworks are there, since they're still in the properties files.

Reply:
I think frameworks are also not required. Pages should be enough

Reply:
Wouldn't it be better to have this consistent with the properties files? Since languages aren't mentioned there IIRC.

Reply:
Kindly only keep `Hugo` as that is the only one not mentioned here: https://github.com/github-starter-workflows/repo-analysis-partner/blob/main/tech_stacks.yml